content/en/security/application_security/enabling/serverless.md

```diff
@@ -216,9 +219,767 @@ To install and configure the Datadog Serverless Framework plugin:
 
 [3]: https://app.datadoghq.com/security/appsec?column=time&order=desc
 
+{{% /tab %}}
+{{% tab "Datadog CLI" %}}
+
+The Datadog CLI modifies existing Lambda function configurations to enable instrumentation without requiring a new deployment. It is the quickest way to get started with Datadog's serverless monitoring.
+
+If you already have tracing enabled on your functions, run the following command with the `--appsec` flag to enable threat detection.
```

Comment:
Can we format this to be a bit more prominent? it's not clear right now that these are the steps to follow if a customer already has APM enabled and they can skip the rest of the steps below